tests/js/spec/views/organizationStream/overview.spec.jsx

```diff
@@ -242,12 +241,16 @@ describe('OrganizationStream', function() {
     });
 
     it('displays the loading icon', function() {
-      wrapper.setState({loading: true});
+      wrapper.setState({savedSearchLoading: true});
```

Comment:
Should we update test name to say what loading icon? Are there two loading icons now?

Reply:
There are 2 loading spinners. One for the whole component, and another for just the issue table.